test/fixtures/workflow_executions.yml

```diff
@@ -193,6 +203,7 @@ irida_next_example_completing_a:
   workflow_url: "https://github.com/phac-nml/irida_next_example_completed"
   run_id: "my_run_id_a"
   submitter_id: <%= ActiveRecord::FixtureSet.identify(:jeff_doe, :uuid) %>
+  namespace_id: <%= ActiveRecord::FixtureSet.identify(:project1_namespace, :uuid) %>
```

Comment:
Starting with here up until line 308, can you change the `project1_namespace` to `projectA_namespace` since that's a project the belongs to the user `jeff doe`

Reply:
Updated in 3b99785

Reply:
Updated in 3b99785